app/javascript/src/stylesheets/shared/select2.scss

```diff
@@ -0,0 +1,5 @@
+.select2 {
+  input {
+    min-width: 250px;
```

Comment:
I had to add this so the placeholder text ("-Select Contact Type Groups-") for contact type groups would fully show. Not a huge fan of it but couldn't find another solution. Open to another idea if anyone has one!